toptal/picasso
# fix(Select): selecting option after search

[SPC-783](https://toptal-core.atlassian.net/browse/SPC-783)

### Description

Fix issue with the `Select` component

### How to test

1. Go to the form select element in the temploy

<img width="662" alt="Screen Shot 2020-12-11 at 14 38 07" src="https://user-images.githubusercontent.com/830217/101935877-7d7b7500-3bbe-11eb-903b-1953fa78943b.png">

2. Search for "4"

<img width="669" alt="Screen Shot 2020-12-11 at 14 38 52" src="https://user-images.githubusercontent.com/830217/101935939-98e68000-3bbe-11eb-9c7d-af4be831bbbe.png">

3. Select one of the options either by clicking on pressing enter
4. Verify that the value is properly selected and the field is focused

<img width="619" alt="Screen Shot 2020-12-11 at 14 39 49" src="https://user-images.githubusercontent.com/830217/101936018-ba476c00-3bbe-11eb-8da3-8775c9832ba5.png">

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Insert screenshots or screen recordings | Insert screenshots or screen recordings |

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>
